examples/5_train_preference_comparisons.ipynb

```diff
@@ -78,7 +74,7 @@
     "    initial_comparison_frac=0.1,\n",
     "    allow_variable_horizon=False,\n",
     "    seed=0,\n",
-    "    initial_epoch_multiplier=2,  # Note: set to 200 to achieve sensible results\n",
+    "    initial_epoch_multiplier=1,\n",
```

Comment:
Do we want to add this back in or is it not necessary? `# Note: set to 200 to achieve sensible results`

Reply:
It's really not necessary, at least for a simple continuous control environment like Pendulum. I think the 200 figure came from the DLFHP paper where it says they trained the reward model for 200 epochs on a random policy _in Atari only_. For continuous control it sounds like they didn't train the reward model at all before starting to train the agent.